### DISCUSSION TITLE
Grow collection size

### DISCUSSION THREAD

**Discussion by mrjod** [nftchef/art-engine#261] [Help + Q&A]
Is there a way to pause and later continue from where it stoped? 

Or if I already have some of the metadata, would it be possible to grow my NFT size , based on those metadata? And not having to start again ?

for instance I want to generate a 5000 collection , 
And I already have 1000 of metadata 
Can i start the continuation from 1001 and not having to start again

**Comment by nftchef** [comment]
Yes, You can continue generating a collection as long as you completed generating and have the `_dna.json` file and backed it up somewhere that is not the `/build` folder, like `build_one`

Then, you can continue using the `--continue` flag in index.js

e.g., 
```
node index.js generate --continue path/to/_dna.json
```
